coralproject/talk
# [CORL-401] Toxicity Score

## What does this PR do?

Displays a toxicity score along with comments that have been reviewed by the perspective API.

## How do I test this PR?

1. Enable the perspective API with an API key
2. Write a comment
3. See the comment in the moderation panel, see the details dropdown
4. Click the details dropdown and see the toxicity score
